### PR TITLE
refactor: v-hide directive and minor cleanup

### PR DESCRIPTION
**Pull request by ferferga**
* Added a ``v-hide`` directive that applies opacity instead of ``display: none``.
* Scope styles of PlayerManager
* Share more global classes

### CODE REVIEW

File: client/layouts/default.vue [hunk 1]
```diff
@@ -130,11 +127,21 @@ interface LayoutButton {
 
 export default Vue.extend({
   mixins: [settingsHelper],
+  props: {
+    keepAliveOptions: {
+      type: Object as () => Record<string, unknown>,
+      default: (): Record<string, unknown> => {
+        return {
+          max: 10,
+          exclude: ['fullscreen-playback']
```
Comment (ferferga) [inline client/layouts/default.vue:136]: We should probably report the fact that this is not working upstream

Comment (ThibaultNocchi) [reply client/layouts/default.vue:136]: What's not working?

**Review by ThibaultNocchi — COMMENTED** [review]
I think you could use Vuetify's `.d-none` instead of adding a Vue directive, wdyt?

**Comment by ferferga** [comment]
d-none and v-show are the same thing, display: none does omit the element in the layout calculations, while ``opacity 0`` and ``visibility hidden`` does not.

File: client/layouts/default.vue (hunk 1 above)
Comment (ferferga) [reply client/layouts/default.vue:136]: The page is not being excluded from the keep-alive. According to Nuxt docs I think our approach for excluding it is right, but it doesn't work.

Comment (ThibaultNocchi) [reply client/layouts/default.vue:136]: I guess you could at least bring that to them yeah

File: client/assets/global.scss [hunk 2]
```diff
@@ -83,10 +83,24 @@ body {
...
+  left: 0;
+  right: 0;
+  bottom: 0;
+}
+
 .pointer {
   cursor: pointer;
 }
 
+.pointer-events-none {
```
Comment (dkanada) [inline client/assets/global.scss:100]: The `pointer-events-none` and `all-pointer-events` classes should match, and also might benefit from being in the same location.

File: client/assets/global.scss [hunk 3]
```diff
@@ -83,10 +83,28 @@ body {
...
+  pointer-events: none;
+}
+
+.all-pointer-events {
```
Comment (dkanada) [inline client/assets/global.scss:104]:
```suggestion
.pointer-events-all {
```

**Review by dkanada — APPROVED** [review]

**Review by MrTimscampi — APPROVED** [review]

**Pull request merged**